validphys2/src/validphys/scripts/vp_multiclosure.py

```diff
@@ -0,0 +1,159 @@
+import sys
+import os
+import logging
+
+#TODO: Look into making these lazy imports
+import prompt_toolkit
+from prompt_toolkit.completion import WordCompleter
+
+from reportengine.compat import yaml
+from reportengine.colors import t
+
+from validphys.app import App
+from validphys.loader import RemoteLoader
+from validphys import compareinconsistentclosuretemplates
+from validphys.promptutils import confirm, KeywordsWithCache
```

Comment:
```suggestion
import sys
import logging

from reportengine.compat import yaml

from validphys.app import App
from validphys import compareinconsistentclosuretemplates
```

These seem all unused